### PR TITLE
Fix system tx gasused

### PR DESCRIPTION
**Pull request by blxdyx**
- fix system tx gasused #367 
- remove bsc checkcompatible

### CODE REVIEW

File: erigon-lib/chain/chain_config.go [hunk 1]
```diff
@@ -615,47 +615,6 @@ func (c *Config) checkCompatible(newcfg *Config, head uint64) *ConfigCompatError
...
-	if incompatible(c.LubanBlock, newcfg.LubanBlock, head) {
-		return newCompatError("luban fork block", c.LubanBlock, newcfg.LubanBlock)
-	}
-	if incompatible(c.PlatoBlock, newcfg.PlatoBlock, head) {
-		return newCompatError("plato fork block", c.PlatoBlock, newcfg.PlatoBlock)
-	}
-	if incompatible(c.HertzBlock, newcfg.HertzBlock, head) {
-		return newCompatError("hertz fork block", c.HertzBlock, newcfg.HertzBlock)
-	}
-	if incompatible(c.HertzfixBlock, newcfg.HertzfixBlock, head) {
```
[the remark this answers is not in the document]
Comment (setunapo) [reply erigon-lib/chain/chain_config.go:656]: 1.user forget to upgrade binary for a hard fork
2.the hard fork passed, but the old binary still can sync, since the hard fork actually has not been triggered, such as HertzFix
3.but once the user upgrade to the latest binary, then `checkCompatible()` will fail, the node will rollback to the hard fork height and sync again.

This PR eliminates this `checkCompatible()` to avoid trigger the rollback in this case.
Basically, I think it is fine, as long as the execution result is same, the result could be accepted, no need to rollback&resync gain.
